docker-compose/oidc/bootstrap.yml

```diff
@@ -0,0 +1,39 @@
+# =================================================================
+# This file defines initial cluster properties for a Redpanda cluster.
+#
+# For more information on bootstrap files, see:
+# https://docs.redpanda.com/current/deploy/deployment-option/self-hosted/manual/production/production-deployment/#configure-a-bootstrap-file
+# =================================================================
+
+#
+# Enable SASL authentication for the Kafka and Admin APIs.
+# https://docs.redpanda.com/current/reference/properties/cluster-properties/#admin_api_require_auth
+admin_api_require_auth: true
+# At least one superuser is required to be able to create other SASL users
+# https://docs.redpanda.com/current/reference/properties/cluster-properties/#superusers
+superusers:
+  - superuser
+  - <oidc-user>
+# https://docs.redpanda.com/current/reference/properties/cluster-properties/#enable_sasl
+enable_sasl: true
+# Enable OIDC (enterprise feature)
+sasl_mechanisms:
+  - 'SCRAM'
+  - 'OAUTHBEARER'
+http_authentication:
+  - 'BASIC'
+  - 'OIDC'
+oidc_clock_skew_tolerance: 600
+oidc_discovery_url: "https://login.microsoftonline.com/<tenant-id>/v2.0/.well-known/openid-configuration"
+oidc_principal_mapping: "$.preferred_username"
+oidc_token_audience: "<client-id>"
+admin_api_require_auth: true
+# Allow topics to be created on first access.
+# https://docs.redpanda.com/current/reference/properties/cluster-properties/#auto_create_topics_enabled
+auto_create_topics_enabled: true
+# Enable data transforms.
+# https://docs.redpanda.com/current/develop/data-transforms/how-transforms-work/
+data_transforms_enabled: true
+# Enable audit logging (enterprise feature).
+# https://docs.redpanda.com/current/manage/audit-logging/
+audit_enabled: true
```

Comment:
they will also need to enable authentication in the Schema Registry and Panda Proxy listeners by setting `      authentication_method: http_basic` 

```
pandaproxy:
  pandaproxy_api:
    - address: 0.0.0.0
      port: 8082
      name: http-proxy
      authentication_method: http_basic
schema_registry:
  schema_registry_api:
    - address: 0.0.0.0
      port: 8081
      name: schema-registry
      authentication_method: http_basic
```

Reply:
in the node config.